docs/getting-started.md

```diff
@@ -8,7 +8,7 @@ ORT:
 * Analyze the dependencies of `mime-types` using the _analyzer_.
 * Scan the source code of `mime-types` and its dependencies using the `scanner`.
 * Run the evaluator to find any rule violations.
-* Generate reports to show the results.
+* Generate reports showing the results.
```

Comment:
Grammar-wise I don't think it's actually *wrong*... it's probably just a bit unusual. Maybe it was done that way to resemble the "to find" one sentence above.

Reply:
I dropped the commit.